server/server.go

```diff
@@ -153,6 +143,15 @@ func StartServer() {
 		srv.Shutdown(serverCtx)
 	}()
 
+	// Add profiling flag to enable profiling routes.
+	if os.Getenv("PPROF_ENABLED") != "" {
+		// pprof attaches routes to default serve mux
+		// host:6061/debug/pprof/
+		go func() {
+			log.Error().Err(http.ListenAndServe(":6061", http.DefaultServeMux))
```

Comment:
Hint/nit: with a `zerolog.Logger` you can call `Err` with an error, and it will return an event with `Error` level: https://github.com/rs/zerolog/blob/c78e50e2da70f4ae63e1b65222c3acf12e9ba699/log.go#L377-L380.

You also *must* call `.Msg`, otherwise the event won't be sent:

```suggestion
			if err := http.ListenAndServe(":6061", http.DefaultServeMux); err != nil {
				log.Err(err).Msg("pprof service returned error")
			}
```